.github/workflows/integration.yml

```diff
@@ -9,6 +9,11 @@ on:
 jobs:
```

Comment:
Do you want to make the equivalent changes to `.github/workflows/snapshot.yml` too? That will build and put an image to quay.io once this merges to main and we can verify that. If all looks good, a follow-up PR to do `.github/workflows/release.yml` will be the last piece.